src/price_data.py

```diff
@@ -587,26 +588,36 @@ def check_database(
                         )
 
                 with sqlite3.connect(db_path) as conn:
-                    query = "SELECT name FROM sqlite_master where type='table'"
+                    query = f"SELECT name FROM sqlite_master where type='table' AND name LIKE '%{config.FIAT_CLASS.name}'"
```

Comment:
Why is `LIKE '%{config.FIAT_CLASS.name}` needed?

Reply:
In the end it only matters that all COIN/FIAT_CLASS data is present, no?
So we only need to check if those pairs are ok.

Reply:
We need to have all prices (if existing) independent of the home fiat.

Reply:
Yeah, just noticed myself. Reverting.